sdk/servicebus/Azure.Messaging.ServiceBus/tests/Processor/SessionProcessorLiveTests.cs

```diff
@@ -1852,7 +1852,7 @@ Task ProcessMessage(ProcessSessionMessageEventArgs args)
                     {
                         Assert.GreaterOrEqual(processor.InnerProcessor._tasks.Count, 20);
                     }
-                    if (ct == 90)
+                    if (ct == 75)
```

Comment:
It took me a few seconds to realize that this was intended to be `count` and not `cancellationToken` and I was confused about the 75 comparison.    We may want to consider being a bit more verbose with naming for clarity.